src/Parsers/ExpressionElementParsers.cpp

```diff
@@ -188,6 +188,20 @@ bool ParserIdentifier::parseImpl(Pos & pos, ASTPtr & node, Expected & expected)
         ++pos;
         return true;
     }
+    else if (pos->type == TokenType::StringLiteral)
```

Comment:
There is a complication: this code allows parsing a string literal as an identifier in every context. It can be undesirable. I expect to allow it only when we expect a table name.